packages/accounts/src/kernel-zerodev/provider.ts

```diff
@@ -7,40 +7,11 @@ import {
 } from "@alchemy/aa-core";
 
 export class KernelAccountProvider extends SmartAccountProvider<HttpTransport> {
-  gasEstimator: AccountMiddlewareFn = async (struct) => {
-    const request = deepHexlify(await resolveProperties(struct));
-    const estimates = await this.rpcClient.estimateUserOperationGas(
-      request,
-      this.entryPointAddress
-    );
-
-    estimates.verificationGasLimit =
-      (BigInt(estimates.verificationGasLimit) * 130n) / 100n;
```

Comment:
@moldy530 

I didn't have accurate answer to your question about requiring 30% extra gas usage for kernel. Turns out, it was because i added an inaccurate dummy signature which leads to underestimation of gas. Fixed in this update